src/linter/lintWorkspace.ts

```diff
@@ -33,7 +36,7 @@ export default async function lintWorkspace(
 		patternsMatch,
 	});
 	reader = resolveReader({
-		patterns: options.ignorePattern ?? [],
+		patterns: ignorePattern ?? [],
```

Comment:
All the information is available already, so something like this would work:

[
		...(config.ignores ?? []),
		...(options.ignorePattern ?? []),
	].filter(($) => $);

Reply:
True, that also works. I though of not duplicating the merging logic. I'll refactor it.

Reply:
Done